### DISCUSSION TITLE
Inverse pole figure issue

### DISCUSSION THREAD

**Discussion by apulgarinc** [mtex-toolbox/mtex#755] [Ask Anything]
how can I do  an inverse pole figure just for the  desired zone in the part that the material pile-up?
![Captura](https://user-images.githubusercontent.com/72534783/107360935-5dc8d600-6ab5-11eb-8a17-1a98f688d6c8.PNG)

**Comment by ralfHielscher** [comment]
Hi you should restrict you EBSD data to this region of interest. 

1. define the region of interest as a polygon
2. restrict to this region as described here: https://mtex-toolbox.github.io/EBSDSelect.html#7
3. plot the inverse pole figure 

I hope this helps,

Ralf.

**Reply by apulgarinc** [reply]
hi,

thanks for the help, now i have another one is this error message
Unable to perform assignment because the size of the left side is 1-by-2 and the size of the right
side is 0-by-0.

Error in selectPolygon (line 21)
    poly(end+1,:) = [xi,yi]; %#ok<AGROW>

Error in test_1 (line 39)
poly = selectPolygon

**Reply by ralfHielscher** [reply]
Unfortunately I can not reproduce this error. Maybe you can simple define the polygon by hand?

Ralf.
